Python/Product/Analysis/Interpreter/Ast/AstPythonInterpreter.cs

```diff
@@ -123,17 +131,37 @@ string name
             var firstBit = i < 0 ? name : name.Remove(i);
             string searchPath;
 
+            ModulePath mp;
+            bool isInvalid, isMissing;
+            string errorParam;
+
             if (packages.TryGetValue(firstBit, out searchPath) && !string.IsNullOrEmpty(searchPath)) {
-                try {
-                    return ModulePath.FromBasePathAndName(searchPath, name);
-                } catch (ArgumentException) {
+                if (ModulePath.FromBasePathAndName_NoThrow(
+                    searchPath,
+                    name,
+                    null,
+                    null,
+                    out mp,
+                    out isInvalid,
```

Comment:
You can now do `out _` for unused out params, to avoid declaring dummy variables.